src/scripts/containers/VideoManagerContainer.js

```diff
@@ -45,15 +45,25 @@ class VideoManagerContainer extends Component<Props, void> {
   }
 
   render () {
+    const uploads = this.props.uploads
     const selectedVideo =
       this.props.selectedVideo && this.props.selectedVideo.id
+    console.log(uploads)
+
+    let videolist = null
+    if (uploads.size !== 0) {
+      videolist = <VideoList onItemClick={this.onVideoListItemClicked} />
+    }
+
     return (
-      <Wrapper>
-        <VideoList onItemClick={this.onVideoListItemClicked} />
-        {selectedVideo === null && <UploadFile />}
-        {selectedVideo !== null && <VideoForm />}
+      <div>
+        <Wrapper>
+          {videolist}
+          {selectedVideo === null && <UploadFile />}
+          {selectedVideo !== null && <VideoForm />}
```

Comment:
I would rewrite this as a ternary so that we don't basically write the logic 2x:

```javascript
{ selectedVideo === null ? <UploadFile /> : <VideoForm /> }
```

Reply:
agree, i'll change this